src/Components/inputs/file.jsx

```diff
@@ -19,8 +17,7 @@ const File = () => {
         id="first-file-input" 
         type="file" 
         onChange={(e) => {
-          getImage(e.target.value)
-          console.log('fireeeeeeeeeee')
+          getImage(e.target.files[0])
```

Comment:
this should just be `onChange={getImage}` then you do the slicing inside `getImage`. the event(e) would be parsed in automatically.